CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added timeout to `DuckDBQuery` and `SAPRFCToDF`
 - Added support for SQL queries with comments to `DuckDB` source
 - Added "WITH" to query keywords in `DuckDB` source
+- Added `avro-python3` library to `requirements`
```

Comment:
I don't think requirements should be ticked, also the general way to say this would be "added x dependency", as some project don't use requirements.txt for example (perhaps also this project in the future).